listenbrainz/db/tests/test_lastfm_user.py

```diff
@@ -58,8 +58,8 @@ def test_user_load_by_id(self):
 
     def test_user_get_play_count(self):
         date = datetime(2015, 9, 3, 0, 0, 0)
-        test_data = generate_data(date, 100, self.user.name)
-        self.assertEqual(len(test_data), 100)
+        test_data = generate_data(date, 5, self.user.name)
```

Comment:
Since our listen_count is lagging in DB. So just after inserting the listens we cant get the exact listen count. Iliekcomputers suggested that if we can't get exact listen_count then we must at least test that we get an int value for sure. So I made it to test that listen_count is of type int. Decreased the generate_data() from 100 to 5. Because we dont need 100 listens to just test the type of listen_count returned.